packages/medusa/src/api/routes/store/orders/index.ts

```diff
@@ -1,4 +1,6 @@
 import { Router } from "express"
+import "reflect-metadata"
```

Comment:
Is this necessary?

Reply:
Seems to be, sometimes it works without it and sometimes it don't. We should investigate `reflect-metadata` as its behaviour seems kind of random half the time